packages/app/pages/dev.tsx

```diff
@@ -53,28 +57,28 @@ export default function DevPanel(): ReactElement {
           <h3>Login</h3>
         </Divider>
         <a href="/api/v1/login/dev?userId=1">
-          <DevPageButton type="default" htmlType="submit">
+          <Button style={{ margin: "15px" }} type="default" htmlType="submit">
```

Comment:
why're we unstyling this?

Reply:
Whoops.. the old version must have slipped in when I merged. Got that fixed up